Create DAQ Subsystem.md

aaronlittleton opened the pull request.

aaronlittleton committed:
Create Create DAQ Subsystem.md

aaronlittleton committed:
Update Create DAQ Subsystem.md

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+## Constraints
+|No.        |Constraints   |Origin   |
+|---|---------------------|--------|
+|3|The DAQ must have at least 10 analog inputs, to cover all possible sensors being used from the sensor integration system.         |Design Constraint |
```
I am wondering why constraint 3 says you need 10 analog inputs. You need to detect rpm and torque. Seems like you would need 2 not 10. Why 10?

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+
+## Analysis
+
+### USB-6215:
```
Could you use something less expensive to minimize cost? May be the USB-6003????

aaronlittleton committed:
Update Create DAQ Subsystem.md

Update to constraint 3.

The reason for the need of a minimum 10 analog inputs was a suggestion from Jesse, as we originally thought we would need more than 2 inputs.  The constraint has been updated

The reason that we have decided to use the USB-6215, compared to other products such as the USB-6003, is due to the fact that it has 4 PWM Outputs, which will be needed in other subsystems.  This product also provides excellent resolution which will allow us to get a precise measure compared to real time.  This product was also approved by Jesse when brought to his attention.

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+|16|Analog input pins must be able to read an output of 0-5 V, 0-10 V, or 4-20mA input from the sensors.       |Derived from further considerations|
+|17|The DAQ must have an analog-to-digital converter with a minimum of 10-bit resolution.       |Derived from further considerations|
+|18|The DAQ must have a sampling rate of at least 4.8kS/s       |Derived from further considerations|
+|19|The DAQ must be able to read the RPM sensor in increments of 1 rpm, voltage in increments of 0.5 V, current in increments of 0.1 A, and torque in incremets of 0.1 N-m.      |Derived from further considerations|
```
Work = Toque * Rotational Speed. Why you are reading voltage and current?

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+
+## Analysis
+
+### USB-6215:
```
Could you use something less expensive to minimize cost? May be the USB-6003????

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+  For voltage, our E is 100 VDC, which allows our precision to be 0.001526 V. This means the logger will respond to voltage changes of 0.001526V or more. 1% of E is 1; therefore precision of 0.001526 meets and exceeds the limitations of constraint 19.
+
+
+For current, our E is 50 A, which allows the precision to be 0.000763 A. 1% of E is 0.5; therefore precision of 0.000763 meets and exceeds the limitations of constraint 19.
```
Why read current?

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+
+  Our E value was chosen as higher than the expected value to prepare for a worst-case scenario. A lower E value would lead to more precision.
+
+  For voltage, our E is 100 VDC, which allows our precision to be 0.001526 V. This means the logger will respond to voltage changes of 0.001526V or more. 1% of E is 1; therefore precision of 0.001526 meets and exceeds the limitations of constraint 19.
```
Why read voltage?

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+For current, our E is 50 A, which allows the precision to be 0.000763 A. 1% of E is 0.5; therefore precision of 0.000763 meets and exceeds the limitations of constraint 19.
+
+
+  For RPM, our E is 1000 RPM, this is based on the max rpm of the rear wheel being 700.69. This means the precision is 0.01526. 1% of E is 10; therefore precision of 0.01526 meets and exceeds the limitations of constraint 19.
```
We are not reading the speed of the back wheel but the speed of the roller. What is the gear ratio between the back wheel and roller? How does this affect your calculation?

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+
+  For RPM, our E is 1000 RPM, this is based on the max rpm of the rear wheel being 700.69. This means the precision is 0.01526. 1% of E is 10; therefore precision of 0.01526 meets and exceeds the limitations of constraint 19.
+
+  For torque our E is 10 Nm, this is based on the max torque of the rear wheel being 4.995 Nm. This means the precision is 0.0001526 Nm. 1% of E is 0.1; therefore precision of 0.0001526 meets and exceeds the limitations of constraint 19.
```
Again, we are not reading the speed of the back wheel but the speed of the roller. What is the gear ratio between the back wheel and roller? How does this affect your calculation?

micahr234 commented on Documentation/Signoffs/Create DAQ Subsystem.md:
```diff
@@ -0,0 +1,109 @@
+|No.        |Constraints   |Origin   |
+|---|---------------------|--------|
+|3|The DAQ must have at least 2 analog inputs, to cover all possible sensors being used from the sensor integration system.         |Design Constraint |
+|16|Analog input pins must be able to read an output of 0-5 V, 0-10 V, or 4-20mA input from the sensors.       |Derived from further considerations|
```
Should you remove 4-20mA if you subsystem will not support that?

micahr234 commented:
Please reply to my comments and let me know how you plan to address or not address each one.

aaronlittleton committed:
Update Create DAQ Subsystem.md

aaronlittleton committed:
Update Create DAQ Subsystem.md

micahr234 reviewed (approved):
I am approving but for cost purposes we need to use something less expensive like the USB-6003. We may have to change the throttle control to receive analog instead of a pwm signal.
